cmd/gaia/app/sim_test.go

```diff
@@ -326,15 +326,16 @@ func TestGaiaImportExport(t *testing.T) {
 	}()
 	newApp := NewGaiaApp(log.NewNopLogger(), newDB, nil)
 	require.Equal(t, "GaiaApp", newApp.Name())
-	request := abci.RequestInitChain{
-		AppStateBytes: appState,
+	var genesisState GenesisState
+	err = app.cdc.UnmarshalJSON(appState, &genesisState)
+	if err != nil {
+		panic(err)
 	}
-	newApp.InitChain(request)
-	newApp.Commit()
+	ctxB := newApp.NewContext(true, abci.Header{})
+	newApp.initGenesis(ctxB, genesisState)
```

Comment:
I needed to call `initGenesis` instead of `InitChain`, since we cannot assert runtime invariants in this test (because they'll fail, since genesis cannot be exported-imported without height modifications, but here we want to assert store equality).

Reply:
thanks